### ISSUE TITLE
[Crash] When launching game after placing prefab overlapping with existing block. 

### ISSUE THREAD

**Issue by Garrett-C** [tsunamayo/Starship-EVO#3917]
### Short Description of the issue:

I placed a prefab and it overlapped with an existing block. Now the save can't be loaded without the game crashing.

### Steps to reproduce the issue:

Place a prefab that overlaps with something.
Reload the save

### Environment
- Starship EVO version: 21w26a
- Graphics Card: RTX 3060

### Additional information:
Here is the save where it happens. 
[Uploading Era of no vox.zip…]()

**Comment by tsunamayo** [comment]
@Garrett-C Hi, so it seems I cant download the blueprint. Do you reproduce the issue consistently? Thanks

**Comment by tsunamayo** [comment]
@Garrett-C Should be fixed in 21w26b. If not ping me and reopen the case! Cheers

**Comment by Garrett-C** [comment]
Ok, so I just reproduced the issue again today. So here's what seems to happen.

1. Place prefab with child entities overlapping blocks.
2. Game no longer lets you place or delete blocks
3. Restart the game
4. Launch the previous character
5. World opens for a second then crash popup appears
6. Game crashes and closes. 

Here is the logfile from when I reopened the world
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/6757240/Player.log)

Here is the new world I used that crashes when opened. Hopefully this one actually uploads properly. (My bad on that last one...)
[Testing123.zip](https://github.com/tsunamayo/Starship-EVO/files/6757245/Testing123.zip)
